app/Global.scala

```diff
@@ -62,5 +62,8 @@ object Global extends GlobalSettings{
       }
     }
   }
+  
+  Some("").get
```

Comment:
![Codacy](http://localhost:9000/versioned/images/favicon.png) Issue found: [Usage of get on optional type.](http://localhost:9000/app/johann_8/executor-tests/file/35/issues/source?bid=13&fileBranchId=15#l66)